airbyte-integrations/connectors/source-orb/source_orb/source.py

```diff
@@ -448,6 +448,37 @@ def path(self, **kwargs) -> str:
         return "plans"
 
 
+class Invoices(IncrementalOrbStream):
+    """
+    Fetches non-draft invoices, including those that are paid, issued, void, or synced.
+    API Docs: https://docs.withorb.com/docs/orb-docs/api-reference/operations/list-invoices
+    """
+
+    @property
+    def cursor_field(self) -> str:
+        """
+        Invoices created in the past may be newly issued, so we store state on
+        `invoice_date` instead.
+        """
+        return "invoice_date"
+
+    def path(self, **kwargs) -> str:
+        return "invoices"
+
+    def request_params(self, stream_state: Mapping[str, Any], **kwargs) -> MutableMapping[str, Any]:
+        request_params = super().request_params(stream_state, **kwargs)
+        # This doesn't make sense for invoices, since the cursor field is `invoice_date`, and the
+        # created_at for an invoice isn't meaningful, so any attempt to use it should be prevented.
+        if "created_at[gte]" in request_params:
```

Comment:
why would "created_at[gte]" be in the request params if the "cursor_field" property is set to "invoice_date" for this stream?

If it's not possible, we should remove this block to limit the cruft

Reply:
`created_at[gte]` is added in the base class, `OrbStream`, based on the `start_date` configuration (see [here](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-orb/source_orb/source.py#L57)), which I can imagine might be relevant for other resources even if the cursor field itself isn't `created_at`. 

If it's preferable based on what you see in other Airbyte connectors, I can refactor the `Invoices` `IncrementalStream` to not take in a `start_date` at all in the constructor, since it's not honored.

Reply:
refactored as above!